roles/edpm_nova/tasks/configure_compute.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Configure compute configs
+  block:
+    - when: edpm_nova_cpu_shared_set|string
+      name: Set CPU shared set
+      community.general.ini_file:
```

Comment:
communit.general is not part of the validated content supported in our downstream distobution fo ansible

https://access.redhat.com/support/articles/ansible-automation-platform-certified-content
and per https://access.redhat.com/articles/4916901
that means that there is no official support for those modules in our product.